extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -51,12 +53,13 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 	 */
 	function pushAd(slot, slotPath, slotTargetingData, extra) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 25 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)